demisto_sdk/commands/init/tests/contribution_converter_test.py

```diff
@@ -618,14 +618,15 @@ def test_format_converted_pack(self, contribution_converter, mocker, new_pack):
         - Running the format_converted_pack method to format the files
 
         Then
-        - Ensure the repo we are comparing with is "xsoar-contrib/master"
+        - Ensure that we format the untracked files as well and the interactive flag is set to false
         """
         contribution_converter.create_new = new_pack
         result = mocker.patch('demisto_sdk.commands.init.contribution_converter.format_manager',
                               side_efect=self.mock_format_manager())
         contribution_converter.format_converted_pack()
 
-        assert result.call_args[1].get('prev_ver') == 'xsoar-contrib/master'
+        assert result.call_args[1].get('include_untracked')
+        assert not result.call_args[1].get('interactive')
```

Comment:
```suggestion
        assert result.call_args[1].get('interactive') == False
```
Better be explicit, `not` might accept `None`.